packages/tests/src/proposeExecute.ts

```diff
@@ -49,10 +50,11 @@ didTestCases.forEach(({ didType, getDidAccount }) => {
       ) =>
         program.methods
           .proposeTransaction(
-            Buffer.from([]), // no controller chain,
+            [], // no controller chain,
```

Comment:
I did not see any changes in the rust code around this and thought Vec<u8> needs to be a Buffer on the client side. Am I missing something?